scarlet/observation.py

```diff
@@ -204,7 +204,7 @@ def match(self, model_frame):
             self.slices = tuple(([slice(s) for s in shape]))
 
             # Now we setup the parameters for the psf -> kernel FFTs
-            shape = np.array(model_frame.psfs.shape) - 1
+            shape = np.array(model_frame.psfs.shape) 
```

Comment:
can you explain this change?

Reply:
@herjy ?

Reply:
Sorry, as I said, I'm away. To be completely honest, I'm not entirely sure as to the fundamental reason here. All I know is that before I did that, the tests gave me a result that was shifted by 1 pixel exactly and now it doesn't. I strongly suspected that this was related to the introduction of the condition on pair-sized psfs. I wonder if this could be linked to our dipole problem.

Reply:
Changes like that are very problematic because we don't understand why they're needed. Can you or @fred3m check if there was implicit padding done before or after this piece is called, which differs in the nD FFT and the 1D FFT.

Btw, I don't think our dipoles come from that: they are much smaller than 1-pixel shifts.